src/backends/policy_engine/souffle/raksha_policy_datalog_emitter.cc

```diff
@@ -31,6 +31,9 @@
 
 ABSL_FLAG(std::string, policy_rules, "", "file containing policy rules");
 ABSL_FLAG(std::string, datalog_file, "", "path to policy directory");
+ABSL_FLAG(std::optional<bool>, skip_declarations, std::nullopt,
```

Comment:
Can you split the changes to `skip_declarations` to a separate PR?

Reply:
I think you can simply roll back https://github.com/google-research/raksha/pull/757 (or the related cl/482484092)

Reply:
#757 and these changes are different. Added skip_declarations arg to auth_logic binary for use cases which use only auth_logic but do not rely on our static dl files (Ex: Oak). Such cases need to have an option to not skip any declarations.